nengo_spinnaker/operators/sdp_receiver.py

```diff
@@ -27,22 +28,18 @@ def make_vertices(self, model, *args, **kwargs):
 
         # Get all outgoing signals and their associated connections (this
         # SHOULD be a 1:1 mapping)
-        out = model.get_signals_connections_from_object(self)
-        for signal, connections in six.iteritems(out[OutputPort.standard]):
-            assert len(connections) == 1, "Expecting a 1:1 mapping"
-            conn = connections[0]
-
+        for signal, tps in model.get_signals_from(self)[OutputPort.standard]:
```

Comment:
I still hate `tps` (counter-intuitively, I'm reviewing backwards) especially as, according to the comment, it's a connection....